src/styles/blocks/header.scss

```diff
@@ -0,0 +1,64 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    height: 40px;
+    display: flex;
+    align-self: center;
+  }
+}
+
+.nav {
+  .list {
+    padding: 0;
+    margin: 0;
+    list-style: none;
+    display: flex;
+    justify-content: flex-start;
+
+    &__item {
+      margin-right: 20px;
+
+      &:last-child {
+        margin-right: 0;
+      }
+    }
+
+    &__link {
+      display: flex;
+      height: 60px;
+      align-items: center;
+      flex-wrap: nowrap;
+      @extend %font;
```

Comment:
Move nav to a separate scss file, and rewrite selectors and nesting according to changes in html.